.travis/install.sh

```diff
@@ -19,19 +30,32 @@ else
     DARWIN="false";
 fi
 
+
 # Do we need to use a Python from pyenv, or will the system one suffice?
 PYENV_PYTHON="";
 
 if [[ "${TOXENV}" == "pypy" ]]; then
     PYENV_PYTHON="pypy-4.0.1";
```

Comment:
Yes; in a separate PR.  My first attempt to fix this was to start by just bumping pypy to get a version that wheels were built against, but that was a mess.